lte/gateway/c/oai/tasks/nas/emm/emm_data.h

```diff
@@ -143,6 +143,14 @@ typedef struct volte_params_s {
       voice_domain_preference_and_ue_usage_setting;
 } volte_params_t;
 
+struct emm_attach_request_ies_s;
```

Comment:
Because we used emm_attach_request_ies_s within new_attach_info_t, 
With inclusion of emm_proc.h resulted into many compilation issue; so did forward declaration